.travis.yml

```diff
@@ -52,12 +53,43 @@ jobs:
                 - vendor/bin/phpunit --verbose || travis_terminate 1
                 - php php-cs-fixer --diff --dry-run -v fix
 
+        - &OSX_TEST_JOB
+            stage: Test
+            os: osx
+            language: generic
+            before_install:
+                # Set up OSX environment
+                - travis_retry brew update
+                - travis_retry brew tap homebrew/homebrew-php
+                - travis_retry brew install $phpver || brew outdated $phpver || brew upgrade $phpver
+                - brew link $phpver
+                - travis_retry wget https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer -O - -q | php -- --install-dir=/usr/local/bin && mv /usr/local/bin/composer.phar /usr/local/bin/composer
+                - travis_retry composer self-update
```

Comment:
Travis don't include anything useful for PHP dev in their OSX builds (in-fact specifying `language: php` flat out breaks things, so idk) – let me know if there's a better solution though!